main.cpp

```diff
@@ -4,7 +4,20 @@ using namespace std;
 
 int main()
 {
-  cout << "Hey Mel!";
+  int size;
+  char character;
+  cout << "Enter a size for your triangle";
+  cin  >>  size;
+  cout << "Enter a character: ";
+  cin  >>  character;
```

Comment:
Can we have some input checking? To ensure this is valid input. Take a look at `cin.good()` and `cin.fail()` methods. This comment also applies to the `cin << size` code as well.